### ISSUE TITLE
Error detected in HDF5 (1.8.14) thread 140354872735680:

### ISSUE THREAD

**Issue by r-renberg** [jts/nanopolish#253]
I am attempting to run the following command:

nanopolish variants --consensus pJAB186_polished.fasta --reads pJAB186_BAdams_raw_reads.fasta --bam pJAB186_albacore_called_to_canu_assembled_sorted.bam --genome pJAB186_canu_raw.contigs.fasta

I get the following error:
##fileformat=VCFv4.2
##INFO=<ID=TotalReads,Number=1,Type=Integer,Description="The number of event-space reads used to call the variant">
##INFO=<ID=SupportFraction,Number=1,Type=Float,Description="The fraction of event-space reads that support the variant">
##INFO=<ID=BaseCalledReadsWithVariant,Number=1,Type=Integer,Description="The number of base-space reads that support the variant">
##INFO=<ID=BaseCalledFraction,Number=1,Type=Float,Description="The fraction of base-space reads that support the variant">
##INFO=<ID=AlleleCount,Number=1,Type=Integer,Description="The inferred number of copies of the allele">
##FORMAT=<ID=GT,Number=1,Type=String,Description="Genotype">
#CHROM	POS	ID	REF	ALT	QUAL	FILTER	INFO	FORMAT	sample
HDF5-DIAG: Error detected in HDF5 (1.8.14) thread 140354872735680:
  #000: H5F.c line 591 in H5Fopen(): invalid file name
    major: Invalid arguments to routine
    minor: Bad value
terminate called after throwing an instance of 'hdf5_tools::Exception'
  what():  : : error in H5Fopen
Aborted (core dumped)

I attempted to "fix" this with the advise given in issue #52 , but it didn't work. HELP!!

I am very new to all things bioinformatics, so please over explain things to me!  Thank you

**Comment by jts** [comment]
Hi 

Could you let me know what steps you ran before this?

Jared

**Comment by r-renberg** [comment]
I ran the following:
`nanopolish index -d /data1/pJAB186_BAdams_MinINON/raw_reads/fast5_files pJAB186_BAdams_raw_reads.fasta`

`bwa index pJAB186_canu_raw.contigs.fasta`

`bwa mem -x ont2d -t 20 pJAB186_canu_raw.contigs.fasta BAdams_pJAB186_albarcore_called_1dsq_reads.fastq > pJAB186_albacore_called_to_canu_assembled.bam`

`samtools sort -@ 20 pJAB186_albacore_called_to_canu_assembled.bam -o pJAB186_albacore_called_to_canu_assembled_sorted.bam`

`samtools index pJAB186_albacore_called_to_canu_assembled_sorted.bam 
pJAB186_albacore_called_to_canu_assembled_sorted.bai`

I am attempting to improve the consensus sequence of a genome that is only 8.5 kb, so this might be causing some issues as well. The very small genome is the reason I didn't do the `nanopolish_makerange.py` step.

**Comment by jts** [comment]
Hi Rebecca,

The output of albacore is fastq but I see you've started from a fasta file. Did you do some preprocessing of your reads before giving them to nanopolish?

Jared

**Comment by r-renberg** [comment]
I originally tried running it with the fastq file and got an error that was along the same lines (HDF5 related). I then went back to the tutorial page and noticed that the -r input was a .fa in this bit of code `nanopolish variants --consensus polished.{1}.fa -w {1} -r reads.fa -b reads.sorted.bam -g draft.fa -t 4 --min-candidate-frequency 0.1`. So I used poretools to extract a fasta file from the albacore called reads. I am currently working everything from the beginning again.
